### ISSUE TITLE
[Bug] When using dropzone and withFile the fileNamer function/class parameter is ignored

### ISSUE THREAD

**Issue by kde99** [Laravel-Backpack/CRUD#5363]
# Bug report

### What I did
Created a form with the following field:
```php
CRUD::field("attachments")->type("dropzone")->withFiles([
            "path" => "attachments/" . date("Y/m"),
            "fileNamer" => UUIDFileNameGenerator::class
        ])->label("My attachments");
```

where the UUIDFileNameGenerator is:
```php
class UUIDFileNameGenerator implements FileNameGeneratorInterface
{

    public function getName(string|UploadedFile $file): string
    {
        if(is_object($file) && get_class($file) == UploadedFile::class)
            return $file->getFilename();

        return Str::uuid() . $this->getExtension($file);
    }

    private function getExtension(string|UploadedFile $file): string
    {
        return is_a($file, UploadedFile::class, true)
            ? $file->extension()
            : Str::after(mime_content_type($file), "/");
    }
}
```
in fact you can use *any* class or function you want even:
```php
"fileNamer" => function($a, $b) {
                dd($a, $b);
            }
```

### What I expected to happen
The file namer to work and create the file names I wanted.

### What happened
Nothing. The class/function provided is ignored and is never used.

### What I've already tried to fix it
Change the type of the field to upload or upload_multiple and now it works, but I want to use the dropzone and not the other 2 options.

### Is it a bug in the latest version of Backpack?
**YES**

### Backpack, Laravel, PHP, DB version
PHP 8.1.24 on windows 10 Pro with Laravel 10.30.1 and BD is Dolt (MySQL compatible).

When I run ```php artisan backpack:version``` the output is:
### PHP VERSION:
PHP 8.1.24 (cli) (built: Sep 27 2023 08:12:38) (ZTS Visual C++ 2019 x64)
Copyright (c) The PHP Group                                             
Zend Engine v4.1.24, Copyright (c) Zend Technologies                    

### LARAVEL VERSION:
10.30.1.0           

### BACKPACK PACKAGE VERSIONS:
backpack/activity-log: 2.0.0
backpack/basset: 1.2.1
backpack/crud: 6.3.0
backpack/devtools: 3.0.0
backpack/generators: v4.0.2
backpack/pro: 2.0.18
backpack/theme-tabler: 1.1.1

**Comment by phpfour** [comment]
Hey @kde99,

Thanks for bringing this to our attention. I can confirm that the bug exists and we have added it to our development backlog. 

We will investigate it further internally and will come back when we have news.

Cheers

**Comment by tabacitu** [comment]
Same issue was reported in https://github.com/Laravel-Backpack/CRUD/issues/5367 using a closure - `fileNamer` doesn't get called.

**Comment by pxpm** [comment]
Hey @kde99 thanks for the report. 

This had been fixed in `backpack/pro 2.0.20` . I just tagged it a couple minutes ago, so it could take a while until it gets available in the private repository. In a few moments you can get the new version with `composer update backpack/pro` and have this issue fixed 🙏 

Let me know if that's not the case and I can re-open it. 

Cheers
